doc/read-the-docs-site/tutorials/english-auction-example.rst

```diff
@@ -0,0 +1,332 @@
+.. highlight:: haskell
+.. _english_auction_tutorial:
+
+Plutus Script for an Auction Smart Contract
+==========================================================
+
+Overview
+------------
+
+In this page we look at the Plutus Tx code for a smart contract that controls the auction of an asset, which can be executed on the Cardano blockchain.
+
+Plutus Tx is a high level language for writing the validation logic of the contract, the logic that determines whether a transaction is allowed to spend a UTXO.
+Plutus Tx is not a new language, but rather a subset of Haskell, and it is compiled into Plutus Core, a low level language based on higher-order polymorphic lambda calculus.
+Plutus Core is the code that runs on-chain, i.e., by every node validating the transaction, using an interpreter known as the CEK machine.
+A Plutus Core program included in a Cardano transaction is often referred to as Plutus script or Plutus validator.
+
+To develop and deploy a smart contract, you'd also need off-chain code for building and submitting transactions, querying for available UTXOs and so on.
+You may also want a frontend for your smart contract for better user experiences.
+The frontend and backend of the off-chain part of smart contracts are not covered in this page.
+
+Before we get to the Plutus Tx code, let's briefly go over some basic concepts, including UTXO, datum, redeemer and script context.
+
+The EUTXO Model, Datum, Redeemer and Script Context
+--------------------------------
+
+UTXO (unspent transaction output) is the ledger model used by some blockchains, including bitcoin.
+A UTXO is produced by a transaction, is immutable, and can only be spent once by another transaction.
+In the classic UTXO model, a UTXO contains a wallet address and a value (e.g., some amount of Ada and/or other tokens), and is identified by the wallet address.
```

Comment:
I know that we use a lot the term "classic UTXO model", but I wonder if the term "original utxo model" is perhaps a better ift. For example we could say before in line 26:

UTXO (unspent transaction output) is the ledger model used by some blockchains, and was originally introduced in bitcoin.

Then we can refer it here as "the original utxo model". Otherwise the reader does not know what the "classic utxo model" refers to

Reply:
`and is identified by the wallet address.` is ambiguous. I don't know how much level of information you want to give to the user but you could say

```
In the original UTXO model, a UTXO contains a wallet address and a value (e.g., some amount of Ada and/or other tokens). Inside a transaction, a UTXO is uniquely identified by the wallet address. At the global blockchain level (a chain of blocks of transactions), the UTXO can be uniquely identified by the transaction id and wallet address.
```
You may leave out the "global blockchain level" sentence because then you have to explain what a transaction id is!

Reply:
Actually what I am saying above is not fully correct, I think you need an extra transaction index, both at the local level and the global level, right?

Reply:
I incorporated your comment but didn't include the sentence about "global blockchain level".